src/color/p5.Color.js

```diff
@@ -446,52 +423,81 @@ var colorPatterns = {
  * </code>
  * </div>
  */
-p5.Color._getFormattedColor = function () {
+p5.Color._parseInputs = function() {
   var numArgs = arguments.length;
-  var mode    = this._renderer._colorMode;
-  var maxArr  = this._renderer._colorMaxes[this._renderer._colorMode];
+  var mode = this._renderer._colorMode;
+  var maxes = this._renderer._colorMaxes;
   var results = [];
```

Comment:
ah I see, I think it's because the results var, when outside of _parseInputs, is getting reused by every call to the function. I tried attaching it to p5.Color, but this doesn't work because the method _parseInputs is called with apply() on the p5 object, so there's no access to this property then. let's leave it for now.